public/content/lessons/2017/eulers-formula-via-group-theory/index.mdx

```diff
@@ -1,9 +1,215 @@
+
+<Figure image="adding_complex.png" video="adding_complex.mp4" />
+
+This collection of all sliding actions on the 2D plane goes by the name “the additive group of complex numbers.” Again, the upshot here is that numbers, even complex numbers, are just one example of a group, and the idea of addition can be thought of as successively applying actions.
+
+## Multiplicative Group for Numbers
+
+Numbers, schizophrenic as they are, also lead a completely different life as a completely different kind of group. Consider a new group of actions on the number line: All ways of stretching or squishing it, keeping everything evenly spaced, and keeping 0 fixed in place.
+
+Yet again, this group of actions has that nice property where we can associate each action in the group with a specific point of the thing it’s acting on. In this case, follow where the point that starts at 1 goes. There is one and only one stretching action that brings that point at 1 to the point at 3, for instance. Namely, stretching by a factor of 3. 
+
+<Figure image="stretch_3.png" width="60%" />
+
+Likewise there is one and only one action that brings that point at 1 to the point at $\frac12$: Squishing by a factor of one half.
+
+<Figure image="squish_half.png" width="60%" />
+
+I like to imagine using one hand to fix 0 in place, as using the other to drag 1 wherever I like, while the rest of the number line does what it takes to stay evenly spaced. In this way, every positive number is associated with a unique stretching or squishing action.
+
+Now notice what composing actions looks like in this group. If I apply the stretch by 3 action, and follow it by the stretch by 2 action, the overall effect is the same as applying a stretch by 6 action, the product of the two initial numbers.
+
+<Figure image="stretch_6.png" width="60%" caption="Composing two stretches/squishes is the act of multiplying. Note that squishing the entire line to zero (i.e. multiplying by zero) is not part of the group because there is no inverse." />
+
+In general, applying one action followed by another here corresponds with multiplication. In fact, the name for this group is “the multiplicative group of positive real numbers.” So ordinary, familiar multiplication is one more example in this very general and far reaching idea of groups and the arithmetic within groups.
+
+We can also extend this idea to the complex plane. Again, I like to think of fixing 0 in place with one hand, while dragging around the point at 1, and keeping everything else evenly spaced while doing so.
+
+But this time, as we drag the number 1 to places off the real number line, we see that our group includes not only stretching and squishing actions, but actions with some rotational component as well.
+
+<Figure image="complex_multiplicative.png" video="complex_multiplicative.mp4" />

```

Comment:
This would be a very cool interactive

Reply:
Oh wow you're totally right, and I don't think that would be very difficult at all to make!